addons/website_form_project/controllers/main.py

```diff
@@ -11,5 +11,8 @@ def insert_record(self, request, model, values, custom, meta=None):
             visitor_partner = visitor_sudo.partner_id
             if visitor_partner:
                 values['partner_id'] = visitor_partner.id
+            # When a task is created from the web editor, if the key 'user_ids' is empty, the user_ids is filled with the odoo bot. We set it to False to ensure it is not.
+            if not values.get('user_ids'):
```

Comment:
We could just check if `user_ids` is in values, no? 🧐 
```suggestion
            if 'user_ids' not in values:
```

Reply:
I dont think so, if we do that, then when the web_form contains some assignee (marc demo, mitchel admin etc), we're gonna delete this input. That's why i'm using the 'get'. That way i can check in one line both if the 'user_ids' is in values and if it is empty or not. It would be better to set the default value of the get to False, though.